src/chapters/style/documentation.typ

```diff
@@ -0,0 +1,63 @@
+#import "/src/util.typ": *
+#import mantys: *
+
+= Documentation <sec:style:docs>
+#wbox[
+  This section documents the syntax of documentation commens, see @sec:api:docs for the contents and purpose of documentation comments.
+]
+
+Documentation for items are placed on comments right before the item, with three forward slashes `///` and a leading space.
+The first leading space and comment tokens are stripped of the line and each line is joined to make up the body of the doc comment.
+The content of doc comments should be simple Typst markup with some extra documentation specific syntax and assumptions about structure.
```

Comment:
It is not clear to which markups are considered _simple_. We should describe it clearly or at least asserts definite part of them. simple Typst markup:
- Heading, bold, and Italic markup.
- inline and blocky raw blocks.
- list.
- plain links, like `https://example.com` or `#link("link")[Content]`.
- blabla...

complex Typst markup:
- custom functions.
- conditions and loops.
- blabla...

Reply:
Yes, that sounds reasonable, I'll add a section explaining that. The main goal is of course that it's easily convertible to Markdown.